BitmovinAnalyticsCollector/Classes/Collector/stateMachines/StateMachineDelegate.swift

```diff
@@ -11,5 +11,6 @@ protocol StateMachineDelegate: class {
     func stateMachine(_ stateMachine: StateMachine, didExitSeekingWithDuration duration: Int64, destinationPlayerState: PlayerState)
     func stateMachine(_ stateMachine: StateMachine, didHeartbeatWithDuration duration: Int64)
     func stateMachine(_ stateMachine: StateMachine, didStartupWithDuration duration: Int64)
+    func stateMachineDidAudioChange(_ stateMachine: StateMachine)
```

Comment:
Why doesn't this method follow the same naming as the others here? Doesn't have a reference to the stateMachine as a method parameter.

Reply:
Same as the `qualityChange` stuff - those are zero-duration events, so the `didblablaWith` wouldn't make sense.